### ISSUE TITLE
Incorrect rendering on arches group

### ISSUE THREAD

**Issue by reelpilot** [smeighan/xLights#1668]
I believe this is related to #1620 where the direction of movement on arches was effected by the layer the effect was placed on. I have found that a single strand effect placed on a group of arches with single line as the render style renders incorrectly still. The way it's currently rendering on all layers is the same as the incorrect first layer rendering before the fix on v38. It's hard to explain clearly, so see attached video for a better idea. 

https://youtu.be/5PzVrYDA3xQ

[Arches.zip](https://github.com/smeighan/xLights/files/3537632/Arches.zip)

**Comment by dkulp** [comment]
This is rendering correctly per the configuration of the models in the Arches group.   Arch 01 is the first in the group so it's nodes are first in the "Single Line" render style.   It's left to right across it, then Arch 02, Arch 03, etc...    Thus, this is correct.   You may want to consider re-ordering the models in the Arches group to reverse them (14 down to 1) which would put them in a more logical "left to right" ordering such the the Single Line render style (and similar styles) would work closer to how you expect.

**Comment by reelpilot** [comment]
I believe you said the bug from the previous issue had been there for a long time. My guess is that the bug enabled my less than ideal group ordering to work. Now that the bug has been fixed my previous group ordering is now problematic. Reversing the group order on the arches fixed the arches, but doing the same for my arch bases caused them to start reversing. I traced that down to the starting location being selected as blue square. Switching to green square on the arch bases makes them work normally. Coincidentally green or blue makes no difference on the aches however. I was using blue square because the pixels are ordered right to left in those props, but with all the added controller configuration settings I can make everything green square and reverse the pixel direction to accomplish the same thing.
